platforms/hyperledger-indy/configuration/roles/create/helm_component/templates/crypto-generate.tpl

```diff
@@ -0,0 +1,30 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: {{ component_name }}
+  annotations:
+    flux.weave.works/automated: "false"
+  namespace: {{ component_ns }}
+spec:
+  releaseName: {{ component_name }}
+  chart:
+    path: {{ gitops.chart_source }}/{{ chart }}
+    git: {{ gitops.git_ssh }}
+    ref: {{ gitops.branch }}
+  values:
+    metadata:
+      name: {{ component_name }}
+      namespace: {{ component_ns }}
+    network:
+      name: {{ network.name }}
+    image:
+      name: {{ component_name }}
+      repository: indy-key-mgmt:latest
```

Comment:
The image won't be on Docker Hub, we have to point to a private repo (configured in network.yaml) and add to it just the image name and version here